index.html

```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-rbsA2VBKQhggwzxH7pPCaAqO46MgnOM80zW1RWuH61DGLwZJEdK2Kadq2F9CUG65"
+      crossorigin="anonymous"
+    />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <main id="main">
+      <div class="container-fluid mt-4 ms-2 me-2 align-text-center">
+        <h1 id="title" class="text-center">Dr. Norman Borlaug</h1>
+        <p class="text-center">The man who saved a billion lives</p>
+
+        <div class="container white" id="img-div">
+          <img
+            id="image"
+            src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg"
+            alt="Dr. Norman Borlaug seen standing in Mexican wheat field with a group of biologists"
+            class="mx-auto d-block"
+          />
+          <figcaption id="img-caption" class="text-center">
+            Dr. Norman Borlaug, third from the left, trains biologists in Mexico
+            on how to increase wheat yields - part of his life-long war on
+            hunger.
+          </figcaption>
+        </div>
+        <section id="tribute-info" class="mt-4">
+          <h3 id="timelines" class="text-center">
+            Here's a time line of Dr. Borlaug's life:
+          </h3>
```

Comment:
Following the HTML standard practice for h1-h6 tags, i suggest you have them in order i.e. h1,h2,...h6. kindly check your code and make sure your h1 tags are sequentially arranged.